task_netcore/task4/Client/src/Constants.js

```diff
@@ -0,0 +1,35 @@
+export const REG_EXPR = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[^\da-zA-Z]).{6,}$/;
```

Comment:
What if you will have a lot of regular expression masks. For example, one more for phone number? Could you give more explicit name for this const? "PASSWORD_VALIDATION_EXP" looks better.